docs/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md

```diff
@@ -15,7 +15,8 @@ It can be used to:
 
 - [extend plugins](/developer-docs/latest/development/plugins-extension.md#extending-a-plugin-s-interface)
 - extend [content-types](/developer-docs/latest/development/backend-customization/models.md) programmatically
-- load some [environment variables](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md).
+- load some [environment variables](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md)
+- register the server part of [custom fields](/developer-docs/latest/development/custom-fields.md) that would be used only by the current Strapi application.
```

Comment:
curious: what's the rule about only adding the last dot in a list?

Reply:
That's a good question! 
I would say that, typically, for finite lists, I should use commas at the end of each list item, and a final period on the last one. The enumeration would thus be considered as a whole sentence, split with different bullet points for easier reading, and that follows the Oxford comma rule: - this, - this, - this, - this, - and that.

In the present case, I intended to give a few examples of what can be done, but I don't consider the list exhaustive. So I wanted to avoid using "this, this, this, or this." to not sound exhaustive. Still, I was probably considering this enumeration as a sentence, hence the period in the end. 
But the more I think about it, the more it sounds confusing 😅 

I'd rather have native English speakers fix my punctuation usage. What do you think, @markkaylor and @StrapiShaun?

Reply:
Oh thank you! I like that idea. 

I sometimes use the text to speech feature and this kind of punctuation is a game changer on the result. It's quite inintelligible  otherwise because it becomes a string of words without pause.

Reply:
Oh, thank you very much for highlighting the text-to-speech. 
In light of this, I believe it could be a good idea to add commas here, then. What do you think?